### PR TITLE
CMD: Use tabwritter on fqdn cache list.

### LINKED ISSUE #7300: `cilium fqdn cache list` output is hard to read
The output needs some type of tabwriter functionality, as the variable length of DNS names results in the spacing between the output of columns to differ.

### PR DESCRIPTION
**Pull request by eloycoto**
Fix #7300

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7318)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

File: cilium/cmd/fqdn.go [hunk 1]
```diff
@@ -119,15 +121,18 @@ func listFQDNCache() {
...
-				lookup.ExpirationTime.String(),
-				strings.Join(lookup.Ips, ","))
-		}
+		return
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 5, 0, 3, ' ', 0)
+	fmt.Fprintln(w, "Endpoint\tFQDN\tTTL\tExpirationTime\tIPs")
+	for _, lookup := range lookups {
+		fmt.Fprintf(w, "%d\t%s\t%d\t%s\t%s\n",
```
Comment (nebril) [inline cilium/cmd/fqdn.go]: Can you add a trailing `\t` before the newline? This way, the last column will also be alligned.

File: cilium/cmd/fqdn.go [hunk 2]
```diff
@@ -119,15 +121,18 @@ func listFQDNCache() {
...
-				lookup.Fqdn,
-				lookup.TTL,
-				lookup.ExpirationTime.String(),
-				strings.Join(lookup.Ips, ","))
-		}
+		return
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 5, 0, 3, ' ', 0)
+	fmt.Fprintln(w, "Endpoint\tFQDN\tTTL\tExpirationTime\tIPs")
```
Comment (nebril) [inline cilium/cmd/fqdn.go]: Can you add a trailing `\t` here? https://play.golang.org/p/e2w1hO5EeJI

**Review by nebril — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
RuntimeFQDNPolicies CNAME follow test failed; https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/10539/testReport/junit/(root)/runtime/RuntimeFQDNPolicies_CNAME_follow/

**Comment by ianvernon** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]

**Comment by joestringer** [comment]
Hit #7105, but this is a CLI change. Clearly unrelated, will merge.

**Pull request merged**